### PR TITLE
chore(nx-dev): switch e2e to playwright

### PR DESCRIPTION
**Pull request by barbados-clemens**
## Current Behavior

Nx repo doesn't have a playwright project to dogfood the plugin for

## Expected Behavior

nx-dev-e2e project uses playwright 

also fixed issue with some pages having multiple h1 tags causing playwright to fail.

## Related Issue(s)

Note: playwright be default is set to run against chromium, firefox, and webkit. We can configure this to only run on 1 browser if so desired, but left running against multiple for now. approx run time in CI is 12 mins for firefox, chrome, webkit

Fixes #

### CODE REVIEW

File: nx-dev/nx-dev/project.json [hunk 1]
```diff
@@ -22,9 +22,8 @@
     },
     "sitemap": {
       "executor": "nx:run-commands",
-      "outputs": ["{options.outputPath}"],
+      "outputs": ["{workspaceRoot}/dist/nx-dev/nx-dev/public"],
```
Comment (barbados-clemens) [inline nx-dev/nx-dev/project.json:25]: ran `nx repair` bc nx wasn't happy with existing outputs

**Review by AgentEnder — APPROVED** [review]
package.json + circleci look fine

File: docs/nx-cloud/reference/release-notes.md [hunk 2]
```diff
@@ -1,137 +1,139 @@
 # `nx-cloud` - Release notes
 
+## Nx Cloud package
```
Comment (isaacplmann) [inline docs/nx-cloud/reference/release-notes.md:3]: Why is this file changed?

File: docs/nx-cloud/private/auth-saml.md [hunk 3]
```diff
@@ -134,7 +134,7 @@ if you are interested.
     1. `SAML_CERT=<your-cert-string-from-above>`
     2. `SAML_ENTRY_POINT=<your-login-url-from-above>`
 
-# Helm config
+## Helm config
```
Comment (isaacplmann) [inline docs/nx-cloud/private/auth-saml.md:137]: Why is this file changed?

**Review by isaacplmann — CHANGES_REQUESTED** [review]
Why are markdown files being changed?

File: docs/nx-cloud/private/auth-saml.md (hunk 3 above)
Comment (barbados-clemens) [reply docs/nx-cloud/private/auth-saml.md:137]: because this creates multiple H1 tags on a page which causes playwright to fail.

File: docs/nx-cloud/reference/release-notes.md (hunk 2 above)
Comment (barbados-clemens) [reply docs/nx-cloud/reference/release-notes.md:3]: adds a sub heading to this section so it's understood the different parts of the page and what the encompass. was recommended to go this way to make a better structure when solving the multiple h1 tags per page issue from Ben.

Comment (barbados-clemens) [reply docs/nx-cloud/reference/release-notes.md:3]: ref convo: https://nrwl.slack.com/archives/CT3CQ2F0D/p1692041097243949?thread_ts=1692040441.689549&cid=CT3CQ2F0D

**Review by isaacplmann — APPROVED** [review]
Looks good

**Review by FrozenPandaz — APPROVED** [review]
